modules/40_techpol/NDCplus/datainput.gms

```diff
@@ -14,7 +14,9 @@ $offdelim
 $onlisting
 ;
 
-p40_TechBound(ttot,all_regi,all_te) = f40_TechBound(ttot,all_regi,"%cm_NDC_version%",all_te);
+p40_TechBound(ttot,all_regi,te) = f40_TechBound(ttot,all_regi,"%cm_NDC_version%",te);
+*** windoffshore-todo
+p40_TechBound(ttot,all_regi,"windon") $ (p40_TechBound(ttot,all_regi,"windon") eq 0) = f40_TechBound(ttot,all_regi,"%cm_NDC_version%","wind");
```

Comment:
Likely as above - if it is an all-wind target, read in for "wind"; if it is a wind-on target, don't apply to all of wind :-)